### PR TITLE
Cleared mypy policy errors after rebase

### PR DESCRIPTION
**Pull request by JustEmrick**
# Description
After rebasing and checking the changes, there were many merge conflicts. I created this new branch and did the same django-stubs install and module and recorrected the errors which were slightly different

### CODE REVIEW

File: chirps/policy/views.py [hunk 1]
```diff
@@ -39,12 +41,13 @@ def dashboard(request):
...
 
 @login_required
-def create(request):
+def create(request: HttpRequest) -> HttpResponse:
     """Render the create policy form.
 
     Args:
         request (HttpRequest): Django request object
     """
+    assert isinstance(request.user, User)
```
Comment (alex-nork) [inline chirps/policy/views.py:50]: Why was this change necessary?

Comment (JustEmrick) [reply chirps/policy/views.py:50]: The specific mypy error I kept getting everywhere you see the assert was `Incompatible type for "user" of "Policy" (got "Union[User, AnonymousUser]", expected "Union[User, Combinable, None]")` Since django's request.user can either be an instance of the User model or an AnonymousUser, it's asserting that it's the User.

Comment (JustEmrick) [reply chirps/policy/views.py:50]: I looked into this a bit more. The summary of my following is:

"if you're using Django's default user model, add the following line to your settings:"

`AUTH_USER_MODEL = 'auth.User'`

Once set, we can import the user model into the codebase with this:
```
from django.contrib.auth import get_user_model
User = get_user_model()
```
Does that seem reasonable?

Comment (JustEmrick) [reply chirps/policy/views.py:50]: To add to that, I confirmed that AUTH_USER_MODEL isn't specified currently in settings.

Comment (JustEmrick) [reply chirps/policy/views.py:50]: The way I understood it was the default user model can be either `User` or `AnonymousUser` so it's expecting to indicate which one. Returning `User` is what I thought it is looking for.

Comment (JustEmrick) [reply chirps/policy/views.py:50]: Thanks for this @zimventures! Yea that is a shame. I'll try to rebase and tackle this again and when I come across that error, I'll reference the shared docs.
